README.md

```diff
@@ -1,27 +1,42 @@
-# WebAssembly Community Group: Design
+# WebAssembly Design
 
-This repository contains documents describing the high-level design of
-WebAssembly and is part of the 
-[WebAssembly Community Group](https://www.w3.org/community/webassembly/).
+> This repository contains documents describing the design and high-level overview of WebAssembly.
 
-The actual WebAssembly specification is being developed in the
-[spec repository](https://github.com/WebAssembly/spec/). High-level design
-discussions should continue to be held in the design repository for now so
-that the spec work can remain focused.
+> The documents and discussions in this repository are part of the [WebAssembly Community Group](https://www.w3.org/community/webassembly/).
 
-**Expect the contents to be in flux: everything is still tentative.**
+## Overview
 
-A good starting point is the [high-level design goals](HighLevelGoals.md),
-followed by the [FAQ](FAQ.md).
+WebAssembly or wasm is a portable, low-level, binary-format abstract syntax tree designed as an efficient target language for compilers.
```

Comment:
Wasm is not only a 'target language for compilers', and this use case should not be given prominence. this use can can be noted from the `low-level' and `performance` description. Please also leave the AST comment, this is very significant, wasm is a source code meant to be written and read, and this needs to be give prominence here. Perhaps add a note that the binary format is a one-to-one reversible encoding of the wasm source code, just to make this very clear.